internal/hcp/registry/types.bucket.go

```diff
@@ -535,6 +538,7 @@ func (bucket *Bucket) HeartbeatBuild(ctx context.Context, build string) (func(),
 					nil,
 					hcpPackerModels.HashicorpCloudPacker20230101BuildStatusBUILDRUNNING,
 					nil,
+					buildToUpdate.Metadata,
```

Comment:
The heartbeat probably shouldn't include metadata, the goal is only to update the state of the build so it doesn't timeout and get marked as failed, no other information should be included.

Reply:
Ditto. @JenGoldstrich